### DISCUSSION TITLE
How can I verify that Garbage Collection is actually running?

### DISCUSSION THREAD

**Discussion by idontsov** [docker/buildx#1763] [Q&A]
The documentation [states](https://docs.docker.com/build/cache/garbage-collection/) that "garbage collection runs periodically and follows an ordered list of prune policies." However, I am unsure of the interval at which garbage collection runs and how I can verify if it is scheduled.

I have used a minimalistic configuration (please see below) to limit the cache to 1Gb, but it doesn't seem to be working. When I run the `docker buildx du` command, it shows reclaimable entries larger than 1Gb that were last accessed 2 hours ago.

Am I missing something?

```
# /etc/docker/daemon.json
{
  "builder": {
    "gc": {
      "enabled": true,
      "defaultKeepStorage": "1GB",
      "policy": [
          {"keepStorage": "1GB", "filter": ["unused-for=1h"]}
      ]
    }
  }
}
```

**Comment by rrauenza** [comment]
Looks like you can see the logs in 

journalctl -u docker | grep gc

(Right now I'm only see that I have errors in my specification)
